.github/workflows/upgrade.yml

```diff
@@ -13,16 +13,17 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
-      - uses: actions/setup-java@v3
+      - uses: actions/setup-java@v4
         with:
           distribution: 'zulu'
           java-version: 17
 
+      - uses: gradle/gradle-build-action@v2
+
       - name: Run Updater
-        uses: gradle/gradle-build-action@v2
-        with:
-          arguments: run --args='..'
-          build-root-directory: updater
+        run: |
+          cd updater
+          ./gradlew run --args='..'
```

Comment:
This could be
```suggestion
        run: ./gradlew -p updater run --args='..'
```

Reply:
Oooops, we forgot that the wrapper file is under updater folder.

Reply:
Ah, got it.